ts/private/npm_repositories.bzl

```diff
@@ -26,10 +26,10 @@ def _http_archive_version_impl(rctx):
         elif "dependencies" in p.keys() and "typescript" in p["dependencies"]:
             ts = p["dependencies"]["typescript"]
         else:
-            fail("key `typescript` not found in either dependencies or devDependencies of %s" % json_path)
```

Comment:
Are the quote changes in this file correct?

Reply:
They are intentional. Backticks for anything that goes into docs and single quotes for CLI messages.